### ISSUE TITLE
Support for connecting to a console port?

### ISSUE THREAD

**Issue by Lamby777** [zellij-org/zellij#2833]
Kinda new to this stuff, so maybe this is the wrong place to ask about this, but is there a way to do something like `screen /dev/ttyUSB0` in zellij? Can't find any docs on it, and it doesn't work on its own. I want to use the zellij interface to configure a switch via a console port, because the keybinds and stuff are nicer. Could just run screen inside a zellij process, but that seems a bit overkill, and you might as well just use screen by itself at that point.

**Comment by imsnif** [comment]
Sadly this is not possible... I think it's not really the problem domain we operate in.

**Comment by Zykino** [comment]
Are you using screen like I would use picocom/minicom/cu? I must admit the rare times I need to connect by serial port I don’t multiplex over it (I feels it is already bad enough as it is I much prefer to SSH when I can).

**Comment by Lamby777** [comment]
> Are you using screen like I would use picocom/minicom/cu? I must admit the rare times I need to connect by serial port I don’t multiplex over it (I feels it is already bad enough as it is I much prefer to SSH when I can).

Yeah, pretty much. I tried using [`tio`](https://github.com/tio/tio) but it didn't work for some reason (said connected but showed 0 output). Sticking with `screen` cuz it "just works" and I need at least something to do classwork with.

also sorry bout the dupe, but at least people saw this one ;-;

**Comment by 13r0ck** [comment]
+1 for this issue. Please!

**Comment by 13r0ck** [comment]
I kinda want to look into implementing this and upstream it. If I were to what is the preferred implementation?

Maybe Ctrl + <p>, <a>, to "attach" a panel to a source. Starting with a tty path, but could also attach to an ssh session?
Any opinions?
@imsnif
